### ISSUE TITLE
[Bug]: Accessibility. VM details are not announced when navigating in the list

### ISSUE THREAD

**Issue by sevapopov2** [VirtualBox/virtualbox#293]
### Version

trunk/main

### Host OS Type

Windows

### Host OS name + version

Version 24H2 (OS Build 26100.6899)

### Host Architecture

x86

### Guest OS Type

other

### Guest Architecture

x86

### Guest OS name + version

Not applicable

### Component

GUI

### What happened?

Virtual machine details are not announced by screenreader when navigating with arrow keys in the list.

### How can we reproduce this?

Preconditions:
Screenreader is running.
There is at least 1 virtual machine created.

Steps to reproduce:

1. Navigate to the virtual machine in a tree.
2. Navigate to virtual machine details list with the tab key.
3. Navigate with up and down arrow keys in the list.

Actual behavior:
Virtual machine details are not announced by screenreader when navigating in the list.

Expected behavior:
Virtual machine details are announced when navigating in the list.

Reproduced on:
NVDA 2025.3.
VirtualBox Version 7.2.3 r170871.

### Did you upload *all* of your necessary log files, screenshots, etc.?

- [x] Yes, I've uploaded all pertinent files to this issue.
